### PR TITLE
BuildAndPush script

### PR DESCRIPTION
**Pull request by peterfelts**
Device Simulation deployments run Docker Compose which pulls images from the azureiotpcs Docker account. This means that if you want to make code changes to any of the Simulation microservices, you'll need to change the containers that your deployment VM is using, and this isn't straightforward (you'll have to SSH in to the deployed VM that runs the Simulation microservices and edit the Docker configuration such that it runs different containers). In order to make this easier for developers, the BuildAndPush script will build all Docker containers and push them to a developer-specified repository.

The BuildandPush script calls in to individual build scripts in the /miscroservice/ directory, for each microservice that is being built.

This will break the dependency that deployed simulations will have on the azureiotpcs Docker repository.

### CODE REVIEW

File: microservices/device-simulation-dotnet/deployment/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "@azure-iot/tsconfig",
```
Comment (pratik11jain) [inline microservices/device-simulation-dotnet/deployment/tsconfig.json:2]: you are taking a dependency on azure iot.
I dont think sim should take this dependency.
you should use some generally available recommended tsconfig package

File: microservices/device-simulation-dotnet/deployment/tslint.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "extends": "@azure-iot/tslint-config/strict",
```
Comment (pratik11jain) [inline microservices/device-simulation-dotnet/deployment/tslint.json:2]: same

File: microservices/device-simulation-dotnet/deployment/tsconfig.json (hunk 1 above)
Comment (peterfelts) [reply microservices/device-simulation-dotnet/deployment/tsconfig.json:2]: Removed. Those files shouldn't have even been in this branch. Good catch.

File: microservices/device-simulation-dotnet/deployment/tslint.json (hunk 2 above)
Comment (peterfelts) [reply microservices/device-simulation-dotnet/deployment/tslint.json:2]: Removed.

**Review by pratik11jain — APPROVED** [review]

**Pull request merged**